src/accelerate/utils/other.py

```diff
@@ -207,6 +213,42 @@ def save(obj, f, save_on_each_node: bool = False, safe_serialization: bool = Fal
         save_func(obj, f)
 
 
+# The following are considered "safe" globals to reconstruct various types of objects when using `weights_only=True`
+# These should be added and then removed after loading in the file
+TORCH_SAFE_GLOBALS = [
+    # numpy arrays are just numbers, not objects, so we can reconstruct them safely
+    np.core.multiarray._reconstruct,
+    np.ndarray,
+]
+
+
+def load(f, map_location=None, **kwargs):
```

Comment:
I see why you chose the name, personally I would prefer something more descriptive though.